From 045e9d66213e114b4dfabd44d2962e84897f873f Mon Sep 17 00:00:00 2001
From: Seppo Yli-Olli <seppo.yliolli@gmail.com>
Date: Sat, 27 Mar 2021 19:36:15 +0200
Subject: [PATCH] Also use config.yml in ldconfig element


diff --git a/elements/extensions/mesa/ldconfig.bst b/elements/extensions/mesa/ldconfig.bst
index c5d2bd6c..b2c18c47 100644
--- a/elements/extensions/mesa/ldconfig.bst
+++ b/elements/extensions/mesa/ldconfig.bst
@@ -1,9 +1,7 @@
 kind: stack
 
-variables:
-  prefix: "/usr/lib/%{gcc_triplet}/GL/default"
-  lib: "lib"
-  debugdir: "/usr/lib/debug"
+(@):
+- elements/extensions/mesa/config.yml
 
 public:
   bst:
-- 
2.30.2

