From 4db616a6bbe375ccb4f0161898da6fa19c7bfab8 Mon Sep 17 00:00:00 2001
From: Seppo Yli-Olli <seppo.yli-olli@iki.fi>
Date: Wed, 20 Jan 2021 23:16:37 +0200
Subject: [PATCH] Patch LLVM11 to be built into right location


diff --git a/elements/extensions/mesa/ldconfig.bst b/elements/extensions/mesa/ldconfig.bst
index c5d2bd6cb..7ae0c3f48 100644
--- a/elements/extensions/mesa/ldconfig.bst
+++ b/elements/extensions/mesa/ldconfig.bst
@@ -1,7 +1,7 @@
 kind: stack
 
 variables:
-  prefix: "/usr/lib/%{gcc_triplet}/GL/default"
+  prefix: "/usr/lib/%{gcc_triplet}/GL/mesa-git"
   lib: "lib"
   debugdir: "/usr/lib/debug"
 
diff --git a/elements/extensions/mesa/llvm11.bst b/elements/extensions/mesa/llvm11.bst
index 12e64b1f2..89b2e07f5 100644
--- a/elements/extensions/mesa/llvm11.bst
+++ b/elements/extensions/mesa/llvm11.bst
@@ -13,7 +13,7 @@ environment:
   LD_LIBRARY_PATH: '%{build-root}/%{build-dir}/lib'
 
 variables:
-  prefix: "/usr/lib/%{gcc_triplet}/GL/default"
+  prefix: "/usr/lib/%{gcc_triplet}/GL/mesa-git"
   lib: "lib"
   debugdir: "/usr/lib/debug"
 
-- 
2.29.2

