From f5cf1751d0b689fce0cb17e534933e1330fa774d Mon Sep 17 00:00:00 2001
From: Seppo Yli-Olli <seppo.yliolli@gmail.com>
Date: Mon, 22 Mar 2021 23:27:05 +0200
Subject: [PATCH] Move libclc to inside mesa-git extension


diff --git a/elements/extensions/mesa/libclc.bst b/elements/extensions/mesa/libclc.bst
index a63fc72c..b6f21766 100644
--- a/elements/extensions/mesa/libclc.bst
+++ b/elements/extensions/mesa/libclc.bst
@@ -9,7 +9,7 @@ build-depends:
 
 variables:
   cmake: cmake "-B%{build-dir}" -Slibclc -G"%{generator}" %{cmake-args}
-  prefix: "/usr/lib/%{gcc_triplet}/GL/default"
+  prefix: "/usr/lib/%{gcc_triplet}/GL/mesa-git"
   lib: "lib"
   debugdir: "/usr/lib/debug"
 
-- 
2.30.2

